crates/bevy_render/src/mesh/shape/mod.rs

```diff
@@ -136,11 +138,19 @@ impl Default for Quad {
 
 impl Quad {
     pub fn new(size: Vec2) -> Self {
-        Self { size, flip: false }
+        Self {
+            size,
+            flip: false,
+            uv_scale: Vec2::splat(1.0),
+        }
     }
 
     pub fn flipped(size: Vec2) -> Self {
```

Comment:
Yes, sure, why not. I decided to call it `tiled` instead of `scaled`, to be clearer about the use case (this is for when you want to "tile the texture" on the quad, not to be confused with some sort of scaling of the mesh or transform)